examples/cifar10.cpp

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 #include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/IR/IR.h"
```

Comment:
In cifar10 I do not see anything which would require IR.h. Could you please paste corresponding compilation errors?